cpp/src/parquet/column_writer.cc

```diff
@@ -157,6 +160,8 @@ inline const T* AddIfNotNull(const T* base, int64_t offset) {
   return nullptr;
 }
 
+constexpr int64_t kHashBatchSize = 256;
```

Comment:
What about moving it closer to line 1238?

Reply:
I move it into `BloomFilterWriterImpl` now